migrations/1758207683.sh

```diff
@@ -0,0 +1,6 @@
+echo "Copy configs for keepassxc as alternative password manager options"
+
+if [[ ! -f ~/.config/keepassxc/keepassxc.ini ]]; then
```

Comment:
Couldn't we just create this if $TERMINAL is keepass? We could then copy it if the user changes to keepass?